src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+class Circle extends Figure {
+    private double radius;
+
+    public Circle(Color color, double radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    @Override
+    public double getArea() {
+       return Math.PI * Math.pow(radius, 2);
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: circle, area: " + getArea()
+                + " sq. units, radius: " + radius
+                + " units, color: " + color);
```

Comment:
Creating a figure, don't pass expressions in the constructor. It's better to create a separate variable for the area and pass it on for better code readability.